Main.java

```diff
@@ -1,15 +1,34 @@
-/**
+import java.util.Scanner;
+
+/** This program will tell you what to say durring FizzBuzz
  *
- * @author 
+ * @author Isaac
  */
 public class Main {
 
   /**
    * The method that is executed when you hit the run button.
+   * 
    * @param args the command line arguments
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
-  }
+    Scanner input = new Scanner(System.in);
+    System.out.println("Please enter a number to play FizzBuzz.");
+    int number = input.nextInt();
+    double number_Fizz = number % 3;
+    double number_Buzz = number % 5;
+    if (number_Buzz == 0 && number_Fizz == 0) {
+      System.out.println("You should say FizzBuzz");
+    } else if (number_Fizz == 0) {
+      System.out.println("You should say Fizz.");
+    } else if (number_Buzz == 0) {
+      System.out.println("You should say Buzz.");
+    } else if (number_Fizz ==1 ) {
+      System.out.println("You should say " + number + ".");
+
```

Comment:
Make sure that you are adding comments to your code. Think of the steps your program makes and turn these into comments. Also, for the last condition, this should always happen if you don't do any of the others. So this can be just a simple else statement. If not, it should be if the remainder is not 0. The reason this fails is because the remainer could be 1 or 2!